app/src/main/java/com/ably/tracking/demo/publisher/ably/DefaultAssetTracker.kt

```diff
@@ -86,6 +89,11 @@ class DefaultAssetTracker(
             ?.launchIn(coroutineScope)
     }
 
+    private fun authorize(): String =
+        runBlocking {
```

Comment:
Why `runBlocking`? Can the function not be a suspend one instead?

Reply:
The auth callbacks are still regular functions in the newest release of AAT :wink: The suspending auth callbacks are [waiting to be released](https://github.com/ably/ably-asset-tracking-android/compare/v1.1.0-rc.6...main#diff-ac4a1615c9fdd0962de94cb1f7e74e8cc02f28b79e9c01d45ca4d5906de9e457L5-R6), so they will be available in the next release :crossed_fingers: